frontend/src/js/api/types.ts

```diff
@@ -542,10 +542,39 @@ export interface EntityInfo {
   semantics: ColumnDescriptionSemantic[];
 }
 
+export interface TimeStratifiedInfoQuarter {
+  quarter: number;
+  values: {
+    [label: string]: string;
+  };
+}
+
+export interface TimeStratifiedInfoYear {
+  year: number;
+  values: {
+    [label: string]: string;
+  };
+  quarters: TimeStratifiedInfoQuarter[];
+}
+
+export interface TimeStratifiedInfo {
+  label: string;
+  description: string | null;
+  columns: {
+    label: string; // Matches `label` with `year.values` and `year.quarters[].values`
+    defaultLabel: string; // Probably not used by us
+    description: string | null;
+    type: ColumnDescriptionKind; // Probably not used by us
```

Comment:
For money this would be relevant to append the proper suffix

Reply:
good point, updated